src/components/Monitoring/index.js

```diff
@@ -1,37 +1,80 @@
 // @flow
 import * as React from 'react';
-import { Chart, Geom, Axis, Tooltip } from 'bizcharts';
-
-const sample_data = [
-  { stream: 'Cat Scan', size: 2400 },
-  { stream: "Driver's License", size: 655 },
-  { stream: 'Health Card', size: 980 },
-  { stream: 'X-Ray', size: 5320 },
-  { stream: 'Other', size: 3642 },
-];
-
-const cols = {
-  size: {
-    tickInterval: 1000,
-    alias: 'Size (KB)',
+import { Card, Col, Layout, Row } from 'antd';
+import { Graph } from 'react-d3-graph';
+import { observer } from 'mobx-react';
+import { observable } from 'mobx';
+
+import CurrentNode from 'components/Monitoring/CurrentNode';
+import _data from './data';
+
+const { Content } = Layout;
+
+const makeData = () => {
+  const data = {
+    nodes: [],
+    links: [],
+    focusedNodeId: 1,
+  };
+
+  _data.forEach(d => {
+    data.nodes.push(d);
+    data.links.push({ source: 1, target: d.id });
```

Comment:
We need to switch this up to instead use the genesis block’s ID, if possible to obtain from MultiChain. Right now I’m using a fake “node” that just has an `id` and that node becomes the main node. For today’s demo, I think we should just use this, I will fix this up later.

Reply:
To see what I’m talking about, look inside `src/components/Monitoring/data.js`.

Reply:
I couldn’t wrap my head around how `api/get_peer_info?blockchainName=XXX` worked initially, so I just went ahead and did this. Now that you’ve subbed in my mock calls in on-boarding to actual calls to the Flask app, I think we can easily modify this to instead set the `source` to the `id` of the node requesting `getpeerinfo`.